cmd/open.go

```diff
@@ -2,16 +2,27 @@ package cmd
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/railwayapp/cli/entity"
 )
 
 func (h *Handler) Open(ctx context.Context, req *entity.CommandRequest) error {
-	projectId, err := h.cfg.GetProject()
+	if len(req.Args) == 0 {
+		fmt.Println("Use railway open to open links to Railway from the CLI. Here's whats we have:")
+	}
+
+	projectCfg, err := h.cfg.GetProjectConfigs()
+	if err != nil {
+		return err
+	}
+	project, err := h.ctrl.GetProject(ctx, projectCfg.Project)
```

Comment:
You should be able to open the docs if the project is not found

<img width="222" alt="image" src="https://user-images.githubusercontent.com/3044853/96622211-8dcf4c00-1301-11eb-94c2-82ff6f8336fa.png">